cmd/pullrequest-init/github_test.go

```diff
@@ -279,14 +301,15 @@ func TestParseGitHubURL(t *testing.T) {
 		fmt.Sprintf("https://github.com/%s/%s/pull/%d/foo", owner, repo, prNum),
 		fmt.Sprintf("github.com/%s/%s/pull/%d/foo", owner, repo, prNum),
 	}
-	for _, u := range good {
+	host := []string{"https://github.com", "https://github.com", "http://github.com", "tacocat://github.com", "https://example.com", "https://github.com", "://",}    
```

Comment:
Also change this to a table driven test.

Reply:
I update it.